schematic/manifest/generator.py

```diff
@@ -1,23 +1,19 @@
-from __future__ import print_function
-
-import pickle
-import os.path
-import json
-import collections
+import os
 import logging
-
-from typing import Any, Dict, Optional, Text, List
+from typing import Dict, List, Tuple
+from collections import OrderedDict
+from tempfile import NamedTemporaryFile
 
 import pandas as pd
 import pygsheets as ps
-import synapseclient
+import json
 
 from schematic.schemas.generator import SchemaGenerator
 from schematic.utils.google_api_utils import build_credentials, execute_google_api_requests
-from schematic.synapse.store import SynapseStorage
-from schematic import CONFIG
+from schematic.utils.df_utils import update_df
+from schematic.store.synapse import SynapseStorage
 
-logger = logging.getLogger(__name__)
+from schematic import CONFIG
 
 
 logging.basicConfig(level=logging.DEBUG)
```

Comment:
- Check if `basicConfig()` is needed for every module. 
- Default to `WARNING` (and `-v` goes to `INFO` and then `DEBUG`).